Examples/Swift/LineStyleLayerExample.swift

```diff
@@ -13,8 +13,9 @@ class LineStyleLayerExample_Swift: UIViewController, MGLMapViewDelegate {
 
         mapView.setCenter(
             CLLocationCoordinate2D(latitude: 45.5076, longitude: -122.6736),
-            zoomLevel: 11,
```

Comment:
Is this change needed for your example?

Reply:
I removed this as it's not needed. Thanks for catching.